Blockzilla/BrowserToolbar.swift

```diff
@@ -16,7 +16,7 @@ class BrowserToolbar: UIView {
         super.init(frame: CGRect.zero)
 
         let background = UIView()
-        background.alpha = 0.95
+        background.alpha = 1
```

Comment:
we can remove the line, because the default value for alpha is 1

Reply:
Done!